src/components/EditImageModal/EditImageModal.test.jsx

```diff
@@ -85,319 +90,707 @@ describe('EditImageModal', () => {
     mockQuerySelector.reset();
   });
 
-  describe('page 1 renders', () => {
-    describe('a modal with', () => {
-      it('a closed modal by default', () => {
-        const modal = getModal(editImageModal);
-        expect(modal.find('.modal')).toHaveLength(1);
-        expect(modal.find('.modal.show')).toHaveLength(0);
-        expect(modal.find('.modal-backdrop.show')).toHaveLength(0);
-      });
+  describe('renders', () => {
+    describe('page 1', () => {
+      describe('a modal with', () => {
+        it('correct initial props', () => {
+          const modal = getModal(editImageModal);
+          expect(modal.prop('open')).toEqual(false);
+          expect(shallowWithIntl(modal.prop('title')).find(WrappedMessage).prop('message')).toEqual(messages.editImageModalInsertTitle);
```

Comment:
Using shallowWithIntl here lets me shallow render the actual title prop (which is a React element) being passed to the modal, so I can assert on that wrapper instead of fishing deeper into the modal, which is Paragon's domain. All I want to test is that the prop is correct. I use this pattern a number of places where we passed React elements as props.